ap_src/ap_app/utils/switch_permissions.py

```diff
@@ -0,0 +1,68 @@
+# Models
+from ..models import (UserProfile)
```

Comment:
The brackets around UserProfile shouldn't be necessary.

Reply:
This was code copy and pasted from elsewhere in the codebase. The brackets will be removed.

Reply:
Fixed with 717b45d